### PR TITLE
Add support for editable text on a11y snapshots

### PR DESCRIPTION
**Pull request by darshanparajuli**
I noticed editable text was missing in the a11y snapshots. This PR simply includes it if both context description and text semantic properties are missing.

I tried updating existing tests, not sure if that's the right thing to do here over adding specific test.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,7 @@
 * Kotlin 2.0.21
 * [Gradle Plugin] Gradle 8.10.2
 * [Gradle Plugin] Android Gradle Plugin 8.4.2
+* Support for editable text in accessibility snapshots
```
Comment (jrodbx) [inline CHANGELOG.md:19]: move to Unreleased.  this section is for an already-released version.

File: paparazzi/src/main/java/app/cash/paparazzi/accessibility/AccessibilityRenderExtension.kt [hunk 2]
```diff
@@ -169,6 +169,7 @@ public class AccessibilityRenderExtension : RenderExtension {
     val mainAccessibilityText =
       config.getOrNull(SemanticsProperties.ContentDescription)?.joinToString(", ")
         ?: config.getOrNull(SemanticsProperties.Text)?.joinToString(", ")
+        ?: config.getOrNull(SemanticsProperties.EditableText)?.text
```
Comment (jrodbx) [inline paparazzi/src/main/java/app/cash/paparazzi/accessibility/AccessibilityRenderExtension.kt:172]: does `joinToString` not make sense for this case?

**Review by jrodbx — APPROVED** [review]

File: paparazzi/src/main/java/app/cash/paparazzi/accessibility/AccessibilityRenderExtension.kt (hunk 2 above)
Comment (darshanparajuli) [reply paparazzi/src/main/java/app/cash/paparazzi/accessibility/AccessibilityRenderExtension.kt:172]: It's just a string, as opposed `List<String>` returned by content description and text.

File: CHANGELOG.md (hunk 1 above)
Comment (darshanparajuli) [reply CHANGELOG.md:19]: oops!

**Comment by colinmarsch** [comment]
Thanks for adding this!

Does this support already work for Views or do we need a similar addition in `View.accessibilityText()`?

**Comment by darshanparajuli** [comment]
> Does this support already work for Views or do we need a similar addition in `View.accessibilityText()`?

@colinmarsch I just tested snapshotting `EditText` and it works as expected (it picks up on the actual text).

**Comment by darshanparajuli** [comment]
Looks like `SemanticsProperties.Error` is also missing. I'm going to include it in this change as well if there are no objections.

**Comment by jrodbx** [comment]
> Looks like `SemanticsProperties.Error` is also missing. I'm going to include it in this change as well if there are no objections.

Mind adding in a separate PR?

**Pull request merged**